schema/Makefile

```diff
@@ -0,0 +1,93 @@
+# The version of the open source schema that this version of the
+# enterprise schema depends on.  Change this as new versions become
+# available.
```

Comment:
I'd change this to say 'No further updates will be made to this schema'.  In fact, I wouldn't object to using the number directly without making it a variable, because it won't be changed.